packages/web.service/src/modules/user/router.ts

```diff
@@ -39,6 +56,15 @@ export const userRouter = Router()
         });
     }
   )
+  .get('/user', (req, res) => {
+    const { page, limit } = req.query;
+    getAllUsersAction(page, limit)
+      .then((users) => res.send(users))
```

Comment:
you're sending `Document` directly which is unexpected. map array to the object you want to send:
```suggestion
      .then((users) => res.send(users.map(({ email, role }) => ({ email, role })))
```